config/initializers/sentry.rb

```diff
@@ -1,8 +1,13 @@
 # frozen_string_literal: true
 
 ActiveSupport.on_load(:active_record) do
+
+  VERSION = Rails.root.join("VERSION")
+
   # DSN is set via SENTRY_DSN in ENV
   Sentry.init do |config|
+    config.enabled_environments = %w[staging production]
+    config.release              = File.read(VERSION).strip.split("/").last if VERSION.exist?
```

Comment:
Could you please format this with rubocop, otherwise LGTM

Reply:
done